src/zzz_od/auto_battle/auto_battle_context.py

```
@@ -174,7 +174,8 @@ def chain_left(self, press: bool = False, press_time: Optional[float] = None, re
         state_records = [StateRecord(e, finish_time)]
         if update_agent:
             # 切换角色的状态时间应该是按键开始时间
-            agent_records = self.agent_context.switch_prev_agent(start_time, False)
+            # 使用正确的连携技逻辑，而不是简单的切换到上一个角色
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)